src/modules/drawing/DrawingModule.vue

```diff
@@ -164,69 +166,17 @@ export default {
     },
     watch: {
         async show(show) {
-            this.isLoading = true
             if (show) {
-                this.drawingState = DrawingState.INITIAL
-                this.isNewDrawing = true
-                this.kmlMetadata = null
-
-                // if a KML was previously created with the drawing module
-                // we add it back for further editing
-                if (this.activeKmlLayer) {
-                    log.debug(`Add current active kml layer to drawing`, this.activeKmlLayer)
-                    this.isNewDrawing = this.activeKmlLayer.adminId ? false : true
-                    this.kmlMetadata = this.activeKmlLayer.metadata
-                    await this.addKmlLayerToDrawing(this.activeKmlLayer)
-                    // Remove the layer to not have an overlap with the drawing from
-                    // the drawing manager. We even remove this if the kml load failed as
-                    // this layer is not selectable.
-                    this.removeLayer(this.activeKmlLayer)
-                }
-                this.isDrawingEmpty = this.drawingLayer.getSource().getFeatures().length === 0
-                this.getMap().addLayer(this.drawingLayer)
+                this.isLoading = true
+                this.isDrawingOpen = true
```

Comment:
Because `show` trigger the closing of the drawing menu which is asynchrone due to the `saveDrawing()` and `addLayer()`. To avoid race condition at startup when we have a KML AdminID where we automatically open the drawing mode, I need to know that the mode is still open and not in a closing state.

Reply:
ok